prober/prober.sh

```diff
@@ -0,0 +1,11 @@
+set -eEuo pipefail
+
+ID_TOKEN=$(curl "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=$AUDIENCE" -H "Metadata-Flavor: Google")
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_token
+
+cat jvs_token | ./jvsctl validate --token -
+
+./jvsctl token --auth-token $ID_TOKEN -e "prober" > jvs_breakglass_token
+
+cat jvs_breakglass_token | ./jvsctl validate --token -
```

Comment:
Do we need to check the output and log the error?

Reply:
Just wondering where will we check the validate output?

Reply:
Due to the screenshot contains internal info, this is explained via chat.

Reply:
Can you add a summary here? I'm also wondering how you check the output?For example, we give a valid token but jvsctl returns status code 200 but said it's not valid, this also means there is something wrong with JVS service. Will we be able to catch it?

Reply:
@capri-xiyue Not sure I understand your example. When the CLI can't validate the given token, it's an error https://github.com/abcxyz/jvs/blob/main/pkg/cli/validate.go#L166-L169

Reply:
Yep, I double checked the code, it will throw error instead of saying it's a success just can't validate.